sprokit/processes/adapters/register_processes.cxx

```diff
@@ -29,9 +29,9 @@
  */
 
 #include <sprokit/processes/adapters/kwiver_processes_adapter_export.h>
-#include <sprokit/pipeline/process_factory.h>
 
 #include <vital/plugin_loader/plugin_loader.h>
+#include <sprokit/pipeline/process_factory.h>
```

Comment:
Eh? Why this change? Shouldn't sprokit headers be kept together?

Actually, shouldn't this be:
```c++
#include "input_adapter_process.h"
#include "output_adapter_process.h"

#include <sprokit/processes/adapters/kwiver_processes_adapter_export.h>
// following newline may or may not be needed/beneficial

#include <sprokit/pipeline/process_factory.h>

#include <vital/plugin_loader/plugin_loader.h>
```
...?